package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "c8",
   "version": "7.13.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
I'm not sure this was an intentional change, so I'm bringing it to your attention. Lock file version 3 drops backwards support for NPM v6 in favor of those from 7 onwards. I'm in favor of moving forward, but only if the step is intentional and the maintainer agrees with dropping support for NPM v6 - the version that's shipped with NodeJS 14 by default.

Reply:
Thank you for pointing that out.  Very good feedback.  I think it's a good idea to migrate to lockfile version 3, but this change was unintentional.  To keep a lockfile version at 2, you essentially have to install packages with the following commands.  

```
$ nvm use default
Now using node v16.19.0 (npm v9.5.0)
$ nvm use 12.22.12
Now using node v12.22.12 (npm v8.19.3)
$ rm -rdf package-lock.json node_modules
$ npm i

added 366 packages, and audited 367 packages in 7s

89 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
$ head package-lock.json
{
  "name": "c8",
  "version": "7.13.0",
  "lockfileVersion": 2,
  "requires": true,
  "packages": {
    "": {
      "name": "c8",
      "version": "7.13.0",
      "license": "ISC",
``` 

On a side note, they may have recently changed the version of npm with the LTS version of NodeJS 14. 

```
$ nvm use 14.21.3
Now using node v14.21.3 (npm v9.5.0)
```

Reply:
I believe another patch changed the lock file version.  Removed my change.